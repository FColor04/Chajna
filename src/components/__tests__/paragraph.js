import React from "react"
import renderer from "react-test-renderer"
import StyledParagraph from "../paragraph"
describe("StyledParagraph", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<StyledParagraph>Test Paragraph</StyledParagraph>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})