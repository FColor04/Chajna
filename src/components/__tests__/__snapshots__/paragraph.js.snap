// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledParagraph renders correctly 1`] = `
<div
  className="sc-bdnylx fHTwrT"
>
  Test Paragraph
</div>
`;
